.pre-commit-config.yaml

```diff
@@ -25,7 +25,7 @@ repos:
 - repo: https://github.com/psf/black
   rev: 21.8b0
   hooks:
-  - id: black
+  - id: black-jupyter
```

Comment:
So this does Black for everything as before *and* notebooks? Then cool 👍.

Reply:
Yep.